sematic/scheduling/kubernetes.py

```diff
@@ -53,6 +53,8 @@
     )
 )
 
+DEFAULT_WORKER_SERVICE_ACCOUNT = "default"
```

Comment:
do we need this default in addition to the one in `Values.yml`?

Reply:
this one yes, since in a testing context you will not have the value configured by Helm.